build.xml

```diff
@@ -147,8 +147,10 @@
   </target>
 
   <target name="run" depends="jar"
-          description="Runs the development version of the program">
-    <java jar="${jar.dir}/${productname}.jar" fork="true"/>
+          description="Gives instructions on how to run">
+          <echo message="To run the demonstration:"/>
+          <echo message="Build using 'ant run'"/>
```

Comment:
This self-referential instruction results in the command `ant run` telling the user to "Build using 'ant run'".